templates/table/02-table/table/table-home/table-home.component.html

```diff
@@ -0,0 +1,59 @@
+<o-form class="fill-form" editable-detail="no" show-header="no">
+  <div fxLayout="column" class="bg-level-1" fxFlexFill>
+    <div class="cards-container">
+      <mat-card *ngFor="let cat of types; trackBy: trackById" [ngClass]="selectedType === cat.id ? 'active type-card' : 'type-card'"
```

Comment:
Ayer comentaba dani de añadir la clase "bg-level-04" para que sea estandar para todos

```suggestion
      <mat-card *ngFor="let cat of types; trackBy: trackById" [ngClass]="selectedType === cat.id ? 'active type-card' : 'type-card'" class="bg-level-04"
```